trailblazer/store/crud/read.py

```diff
@@ -30,6 +31,15 @@ def get_nr_jobs_with_status_per_category(
         categories = categories.group_by(Job.name).all()
         return [{"name": category.name, "count": category.count} for category in categories]
 
+    def get_analysis_with_id(self, analysis_id: int) -> Optional[Analysis]:
+        """Get a single analysis by id."""
+        return apply_analysis_filter(
+            filter_functions=[AnalysisFilter.FILTER_BY_ID],
+            analyses=self.get_query(table=Analysis),
+            analysis_id=analysis_id,
+        ).first()
+        return self.Analysis.query.get(analysis_id)
```

Comment:
I this second return supposed to be here?

Reply:
Fixed